### ISSUE TITLE
Reducing prognostic run dependence on fv3fit.keras?

### ISSUE THREAD

**Issue by nbren12** [ai2cm/fv3net#1396]
In principle, tensorflow's saved_model format should be possible to load with tensorflow alone. This allows training a model with custom code in a notebook, saving it, and then applying it online. This assumes that all useful information (e.g. normalization, stacking, etc), has been baked into the keras model/tensorflow graph.

I know that @frodre emulation code for instance does not require that `fv3fit` be installed for prognostic runs. How close to this are we in practice?

cc @mcgibbon @AnnaKwa

**Comment by nbren12** [comment]
This issue is driving at the backwards-compatibility of `fv3fit.load` with saved artifacts from past. The current format is pretty backwards-compatible now that the multi-index isn't saved separately.
